pkg/reconciler/pipelinerun/pipelinerun_test.go

```diff
@@ -6001,10 +5993,11 @@ spec:
       persistentVolumeClaim:
         claimName: $(tasks.a-task.results.aResult)
 `)},
-			disableAA: true,
 			expectedTr: mustParseTaskRunWithObjectMeta(t,
-				taskRunObjectMeta("test-pipeline-run-variable-substitution-b-task", "foo",
-					"test-pipeline-run-variable-substitution", "test-pipeline", "b-task", false),
+				taskRunObjectMetaWithAnnotations("test-pipeline-run-variable-substitution-b-task", "foo",
+					"test-pipeline-run-variable-substitution", "test-pipeline", "b-task", false, map[string]string{
+						"pipeline.tekton.dev/affinity-assistant": "affinity-assistant-0358aabfa2",
```

Comment:
Since the previous setting of `disable-affinity-assistant` was true, the default behavior was not to add the relevant annotations. Now that the feature switch has been removed, the behavior has changed somewhat.

Theoretically, the previous effect can also be achieved by setting other feature switches.

Reply:
has this been documented ? 
1. The behavior change 
2. what the previous effect is and how to achieve it be achieved with other feature switches ? 

This would be good to have in the release notes as well